typescript/introduction/tsc-basic-types/tsc-enum.md

```diff
@@ -3,9 +3,7 @@ author: kapnobatai136
 type: normal
 category: must-know
 links:
-  - >-
-    [Enum
-    Type](typescriptlang.org/docs/handbook/basic-types.html#enum){documentation}
+  - '[Enum Type](https://www.typescriptlang.org/#enum){documentation}'
```

Comment:
this link doesn't lead to the `enum` page

```suggestion
  - '[Enum Type](https://www.typescriptlang.org/docs/handbook/basic-types.html#enum){documentation}'
```

Reply:
Yeah, I have mistakenly copied the wrong one, thanks!